examples/python/ToF/tof_rvc4_exposed_settings.py

```
@@ -0,0 +1,450 @@
+        "  phaseUnwrapErrorThreshold={}\n"
+        "  pipeType={}".format(
+            cfg.enableBilateralFilter, cfg.bilateralStdFactor, cfg.bilateralKernelSize,
+            cfg.enableTemporalNoiseReduction, cfg.tnrMaxGain, cfg.tnrStdFactor,
+            cfg.enableFlyingPixelFilter, cfg.flyingPixelDepthThreshold,
+            cfg.flyingPixelMinDepthOccurrence, cfg.phaseUnwrapErrorThreshold,
+            cfg.pipeType,
+        )
+    )
+
+
+def main() -> int:
```

Ruff (0.15.21)

[warning] Too many branches (16 > 12)

(PLR0912)